atlp-rwanda/e-commerce-ones-and-zeroes-bn
# [Finishes #187355085] Eslint configuration

## What does this PR do?
Configure ESLint and Git Pre commit hook to run tests and ESlint format
## Description of Task to be completed?
This task was about to run Eslint, test, and format before the pre-commit
## How should this be manually tested?
You need to follow the common practice of pushing to the remote repositiory, by git add, git commit -m 'Add message' after that the Eslint will run lint and format to fix the indentation issue and make sure that the code follow ES6 guidelines
## Any background context you want to provide?
## What are the relevant pivotal tracker/Trello stories?
Configure ESLint and Git Pre commit hook to run tests and ESlint format https://www.pivotaltracker.com/story/show/187355085
## Screenshots (if appropriate)
![Screenshot 2024-04-05 160144](https://github.com/atlp-rwanda/e-commerce-ones-and-zeroes-bn/assets/117368635/640dc687-64e1-40fe-a13f-60f90d85c421)

## Questions
